src/_pytest/deprecated.py

```diff
@@ -89,7 +89,10 @@
 
 WARNS_NONE_ARG = PytestRemovedIn8Warning(
     "Passing None to catch any warning has been deprecated, pass no arguments instead:\n"
-    " Replace pytest.warns(None) by simply pytest.warns()."
+    "Replace pytest.warns(None) by simply pytest.warns().\n"
```

Comment:
I feel like we should remove the "pass no arguments" suggestion entirely - clearly this solution is only correct in a fraction of cases, and with that, people will probably not look at the docs to find the proper fix for their situation...